website/docs/config_guide.md

```diff
@@ -1124,3 +1124,90 @@ For example:
   }
 }
 ```
+
+### Configuration callbacks
+
+We use jexl (see https://github.com/TomFrost/Jexl) for defining configuration
+callbacks. This is more limited than allowing arbitrary javascript in our
+configs, but helps prevent things like XSS.
```

Comment:
can probably just leave out the "this is more limited" sentence, I don't see why we have to justify our design decision in the manual